js/main.js

```diff
@@ -1553,22 +1555,24 @@ class App extends React.Component {
 
         return (
           <div key={pane.id} className={isVisible ? '' : 'hidden-window'}>
-            <Comp
-              {...pane}
-              key={pane.id}
-              onClose={this.closePane}
-              onFocus={this.focusPane}
-              onInflate={this.onInflate}
-              isFocused={pane.id === this.state.focusedPaneID}
-              w={panelayout.w}
-              h={panelayout.h}
-              width={this.w2p(panelayout.w)}
-              height={this.h2p(panelayout.h) - PANE_TITLE_BAR_HEIGHT}
-              appApi={{
-                sendPaneMessage: this.sendPaneMessage,
-                sendEmbeddingPop: this.sendEmbeddingPop,
-              }}
-            />
+            <ReactResizeDetector handleWidth handleHeight>
```

Comment:
As a question of curiosity and for the record - was the RGL `onResize` event not sufficient to handle this kind of event? What does the new dependency grant us?

Reply:
The `onResize` event seemed to apply to the entire Grid Layout itself. I was more interested in detecting individual `onResize` events for each of the `<div>` elements, and triggering a re-render for individual elements only on resize. Plus the new dependency offers a really clean solution as we don't have to set up an `EventListener` and attach it to the corresponding DOM Element for each item (it's only 2 lines of extra code).